organizing_hub/templates/calls/callcampaign_detail.html

```diff
@@ -141,11 +144,21 @@ <h3>Callers</h3>
       <p>
         <strong>{{ campaign.callers.all|length }}</strong>
         callers on this campaign.
-        <a href="todo">
-          View
+        <a href="{% url 'organizing-hub-call-campaign-update' campaign.uuid %}">
+          Edit
         </a>
       </p>
 
+      {% include "calls/partials/call_campaign_callers_explainer.html" %}
+
+      {% if campaign.callers.all|length > 0 %}
```

Comment:
do we need empty list logic?

Reply:
What do you mean exactly? This will only show if there are callers. We'll show there are 0 callers on this campaign above and there will be an edit button.

Reply:
I'm just asking, sometimes we have conditional content for empty list.

Reply:
Gotcha, no I think UI makes sense here since we display number of callers.